aries-site/src/examples/templates/forms/SignInExample.js

```diff
@@ -176,6 +221,18 @@ export const SignInExample = () => {
                 label="Remember me"
               />
             </FormField>
+            {credentialError && (
+              <Box
+                direction="row"
+                margin={{ top: 'medium', bottom: 'medium' }}
+                gap="xsmall"
+                round="4px"
+                pad="small"
+                background={{ light: '#FC61613D', dark: '#C54E4B5C' }}
```

Comment:
These look like they are specifying the error background colors, correct? 

Looks like these should be added to the theme so that we are not promoting use of custom HEX values/colors in the Design System.  Can you add an issue for "Add validation error background colors to theme"? Should include: 
- new color namespace - reference: https://design-system.hpe.design/foundation/color#input-palette
- apply new namespace here: https://github.com/grommet/grommet-theme-hpe/blob/1e352af3ec70f5ab3ebcdbaa55a74ddca0f536c1/src/js/index.js#L507
- apply new namespace in SignInExample.js

Reply:
yes! @taysea and I were just saying we did not want to hard code these they should be pulled from theme!